Bin/Client.php

```diff
@@ -68,27 +67,30 @@ class Client extends Console\Dispatcher\Kit {
     /**
      * The entry method.
      *
-     * @access  public
      * @return  int
      */
-    public function main ( ) {
-
+    public function main()
+    {
         $server = '127.0.0.1:8889';
 
-        while(false !== $c = $this->getOption($v)) switch($c) {
+        while (false !== $c = $this->getOption($v)) {
+            switch ($c) {
+
```

Comment:
This blank line is it removed by your lasts commits on `hoa devtools:cs` ?
eg: https://github.com/Hywan/Devtools/blob/9a0d3617e5cd08c4f0b4ac8cac80564912d8c3fd/Bin/Cs.php#L77-78